### ISSUE TITLE
NVDA cannot do read all properly in libre office writer

### ISSUE THREAD

**Issue by burakyuksek** [nvaccess/nvda#8531]
### Steps to reproduce:
1. Open libre office writer, with a very large document with like 200 pages.
2. Come to the top of text
3. Try to perform read all
### Actual behavior:
NvDA reads a few lines of the document, then stops when performed read all
### Expected behavior:
NvDA reads all of the document when performed read all
### System configuration:
Windows 7 64-bit with libre office 6.0
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
2018.2.1
#### Windows version:
7
#### Name and version of other software in use when reproducing the issue:
6.0
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by vsfoote** [comment]
Yes, open as #4119 

https://bugs.documentfoundation.org/show_bug.cgi?id=91739

**Comment by ehollig** [comment]
Closing as duplicate of #4119
